### PR TITLE
feat: type hint `Sink.vc`

### PR DESCRIPTION
**Pull request by VincentRPS**
## Summary

Type hints `Sink.vc` to the `VoiceChannel` class.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

File: discord/sinks/core.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 import time
 import io
 from ..types import snowflake
+from ..channel import VoiceChannel
 
 from .errors import SinkException
```
Comment (krittick) [inline discord/sinks/core.py:34]:
```suggestion
from typing import TYPE_CHECKING
from .errors import SinkException

if TYPE_CHECKING:
    from ..channel import VoiceChannel

```

**Review by krittick — APPROVED** [review]

**Review by krittick — APPROVED** [review]

**Review by Lulalaby — APPROVED** [review]

**Comment by XXMA16** [comment]
`Sinks.vc` is not a `discord.VoiceChannel`, it is a `discord.VoiceClient` as shown in this [example](https://guide.pycord.dev/voice/receiving) by adding a `print(type(sink.vc))` under `once_done`.

**Comment by Lulalaby** [comment]
Stay our of this pr. It's literally a year old lol.

**Comment by Lulalaby** [comment]
Open an issue if that is still a problem. It might've changed over the time tho

**Pull request merged**